spring-boot-samples/spring-boot-sample-cache/src/main/resources/infinispan.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <infinispan xmlns="urn:infinispan:config:7.2">
 
-	<cache-container default-cache="default">
+	<cache-container default-cache="countries">
```

Comment:
Sorry I don't understand. Isn't the `local-cache` below creating a cache named `countries`? `countries` is just one cache used in the application, it's not the default cache.